core/page-assets/loader.php

```diff
@@ -56,7 +56,11 @@ public function enable_assets( array $assets_data ) {
 
 		foreach ( $assets_data as $assets_type => $assets_list ) {
 			foreach ( $assets_list as $asset_name ) {
-				$this->assets[ $assets_type ][ $asset_name ]['enabled'] = true;
+				if ( 'scripts' === $assets_type ) {
+					wp_enqueue_script( $asset_name );
+				} else {
+					wp_enqueue_style( $asset_name );
+				}
```

Comment:
@KingYes Should this be behind the experiment check or not?

Reply:
This is fixed about the old code.